agent/ansible/pbench/agent/roles/epel_repo_install/README.md

```diff
@@ -0,0 +1,34 @@
+epel_repo_install
+=================
+
+This role installs a repo file for EPEL.
+
+Requirements
+------------
+
+Role Variables
+--------------
+
+Dependencies
+------------
```

Comment:
Do we really need all this boilerplate if it's just taking up space? Or is there something useful we could put in these sections. The `pbench_firewall_open_ports` version seems like an improvement.

Reply:
Some roles use some/all of these (e.g. the firewall role). These files are used by Galaxy to provide its views and given how picky it is, I wasn't about to experiment too much: I don't remember if you can omit them when empty. Having them in the file doesn't bother me much: I find it helpful to just copy a file from one role to another and having all the fields there (misspelling "Requirements" would probably cause a Galaxy conniption and those tend to be dangerous :-)  - have you seen my [wallpaper](https://apod.nasa.gov/apod/ap100507.html)? But I suppose that we can add "None" in all of the empty places. I'll experiment a bit at some point and try to figure out the Galaxy rules.

Reply:
See issue #2595